<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeepoch</class>
 <widget class="QWidget" name="timeepoch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lTitle">
     <property name="text">
      <string>Compute the corresponding epoch timestamp</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="leMonth"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="leYear"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lYear">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lMonth">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="leDay"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lDay">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lEpoch">
       <property name="text">
        <string>Epoch timestamp:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leOutputTimestamp">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pbComputeEpoch">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lTitle2">
     <property name="text">
      <string>Compute the date from the corresponding epoch timestamp</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="leTimestamp"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lDate">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lTimeStamp">
       <property name="text">
        <string>Epoch timestamp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leOutputDate">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pbComputeTimestamp">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pbCancel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leDay</tabstop>
  <tabstop>leMonth</tabstop>
  <tabstop>leYear</tabstop>
  <tabstop>leOutputTimestamp</tabstop>
  <tabstop>pbComputeEpoch</tabstop>
  <tabstop>leTimestamp</tabstop>
  <tabstop>leOutputDate</tabstop>
  <tabstop>pbComputeTimestamp</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
